src/components/ConversationItem/index.js

```diff
@@ -24,6 +25,11 @@ const ConversationItem = ({ conversation, onSelect }) => {
         </View>
       </View>
       <Image source={{ uri: topicIcon }} style={styles.topicIcon} />
+      {unreadMessages > 0 && (
```

Comment:
does this work?  `{unreadMessages && (....`

Reply:
In that case you need to do: `{ !!unreadMessages && (....`